resize.js

```diff
@@ -103,7 +103,15 @@ function splitUrl(url) {
   return url.match(/^\/(.*)_(\d+)_(\d+)(_(\d+)x)?\.(.*)/);
 }
 
-function getImageParams(url) {
+function getImageParams(req) {
+  var url = req.url;
+  var queryParams = null;
+  var split = url.indexOf('?');
```

Comment:
There is https://nodejs.org/api/url.html#url_url_parse_urlstr_parsequerystring_slashesdenotehost